docs/agents/sematext-agent/kubernetes/installation.md

```diff
@@ -240,6 +240,8 @@ spec:
         key: node-role.kubernetes.io/master
 ```
 
+_[Read](../permission-requirements.md#container-setups) more about why Sematext Agent needs access to host' files and directories._
```

Comment:
_[Read](../permission-requirements.md#container-setups) more about why Sematext Agent needs access to host' files and directories._

=>

_[Read more](../permission-requirements.md#container-setups) about why Sematext Agent needs access to host files and directories._